### PR TITLE
[Tests] Replace `sklearn_classifier` with `auto_trainer` in tests and examples

### PR DESCRIPTION
**Pull request by Eyal-Danieli**
`sklearn_classifier` from the functions marketplace is no longer supported and might not be working following the release of `scikit-learn` v1.2. In this PR, we update the system tests to use the `auto_trainer` function (also from the functions marketplace - https://github.com/mlrun/functions/blob/master/auto_trainer/auto_trainer.py) instead of the `sklearn_classifier`.

### CODE REVIEW

**Review by guy1992l — APPROVED** [review]

**Comment by AlonMaor14** [comment]
please change here https://github.com/mlrun/mlrun/blob/development/tests/system/projects/test_project.py#L89 as well

**Comment by guy1992l** [comment]
> please change here https://github.com/mlrun/mlrun/blob/development/tests/system/projects/test_project.py#L89 as well

Its not affecting the test you've mentioned, we saw it. Anyway Eyal will do so.

**Comment by AlonMaor14** [comment]
Please check `tests/system/demos/sklearn/test_sklearn.py` as `sklearn_classifier` is being used there as well.
Also, if this function is now obsolete maybe we want to go over our docstrings because it's being mentioned many times as an example.
For example in `import_function`
```
def import_function(url="", secrets=None, db="", project=None, new_name=None):
    """Create function object from DB or local/remote YAML file

    Functions can be imported from function repositories (mlrun Function Hub (formerly Marketplace) or local db),
    or be read from a remote URL (http(s), s3, git, v3io, ..) containing the function YAML

    special URLs::

        function hub: hub://{name}[:{tag}]
        local mlrun db:       db://{project-name}/{name}[:{tag}]

    examples::

        function = mlrun.import_function("hub://sklearn_classifier")
        function = mlrun.import_function("./func.yaml")
        function = mlrun.import_function("https://raw.githubusercontent.com/org/repo/func.yaml")
```

File: tests/system/projects/assets/kflow.py [hunk 1]
```diff
@@ -41,22 +41,22 @@ def kfpipeline(model_pkg_class=default_pkg_class, build=0):
     )
 
     # train the model using a library (hub://) function and the generated data
-    train = funcs["train"].as_step(
+    train = funcs["auto_trainer"].as_step(
```
Comment (AlonMaor14) [inline tests/system/projects/assets/kflow.py:44]: Can you add a comment that the default handler is `train`? it wasn't clear to me at 1st sight

**Review by AlonMaor14 — APPROVED** [review]
LGTM
@guy1992l @Tankilevitch please review so we can merge this (needed for a lot of system tests
@Eyal-Danieli can you please also update https://github.com/mlrun/project-demo - it needs to have the same changes and it is also used in many system tests

File: mlrun/projects/operations.py [hunk 2]
```diff
@@ -87,26 +87,26 @@ def run_function(
 
     example (use with function object)::
 
-        function = mlrun.import_function("hub://sklearn_classifier")
-        run1 = run_function(function, params={"data": url})
+        function = mlrun.import_function("hub://auto_trainer")
+        run1 = run_function(function, params={"label_columns": LABELS}, inputs={"dataset": DATA_URI})
```
Comment (guy1992l) [inline mlrun/projects/operations.py:91]: For the uri I would give a more random sample in the length of an actual uri.
```suggestion
        run1 = run_function(function, params={"label_columns":["target"]}, inputs={"dataset": "xxxxxxxxxx"})
```

**Review by guy1992l — CHANGES_REQUESTED** [review]
tiny change

File: mlrun/projects/operations.py [hunk 3]
```diff
@@ -87,8 +87,12 @@ def run_function(
 
     example (use with function object)::
 
+        LABELS = "is_error"
+        MODEL_CLASS = "sklearn.ensemble.RandomForestClassifier"
+        DATA_PATH = "v3io:///bigdata/bla.parquet"
```
Comment (guy1992l) [inline mlrun/projects/operations.py:92]:
```suggestion
        DATA_PATH = "s3://bigdata/data.parquet"
```

**Review by guy1992l — CHANGES_REQUESTED** [review]
bla is not a cool doc string word :) also always try to make it readable to CE users as well (no v3io).

File: mlrun/projects/operations.py (hunk 3 above)
Comment (Eyal-Danieli) [reply mlrun/projects/operations.py]: Got it. Fixed.

**Review by AlonMaor14 — APPROVED** [review]

**Pull request merged**